internal/metrics/response_writer.go

```diff
@@ -7,31 +7,13 @@ import (
 	"net/http"
 )
 
-// statusResponseWriter captures the response status
-type statusResponseWriter struct {
+// websocketResponseWriter
+type websocketResponseWriter struct {
 	http.ResponseWriter
```

Comment:
@peterbroadhurst this didn't work as expected because the inherited `ResponseWriter` for `websocketResponseWriter` ended up being a `statusResponseWriter` which does not implement `Hijacker`.

I tried climbing the "inheritance stack" with something like the following since `statusResponseWriter` is private:
```golang

type wrapperResponseWriter struct {
   http.ResponseWriter
}
// ...
func (w *websocketResponseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
	h, ok := w.ResponseWriter.(wrapperResponseWriter).ResponseWriter.(http.Hijacker)
        // ...
}
```

but that didn't work either so decided it was easier to fork the library with the fix and have FireFly use that instead.